### PR TITLE
Task #5256 - User like book

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/5256?issue_count=27&issue_position=4&next_issue_id=5250&prev_issue_id=5258

### CODE REVIEW

**Comment by taylq** [comment]
ready

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class BooksController < ApplicationController
   before_action :find_book, except: %i(index)
   before_action :status_follow_book, only: %i(show)
```
Comment (huuphuc93) [inline app/controllers/books_controller.rb]: before_action :status_follow_book, :status_like_book, only: %i(show)

**Comment by huuphuc93** [comment]
commented

**Comment by taylq** [comment]
ready

**Pull request merged**